### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by solomoniosif**
Am creat clasa Car.

### CODE REVIEW

File: car_app.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Car:
+    pass
```
Comment (vvdanila) [inline car_app.py]: There is no new line at the end of the file. Please add it.

Comment (vvdanila) [inline car_app.py]: Please add the class methods.

**Review by vvdanila — CHANGES_REQUESTED** [review]
Need improvements

**Review by victoriabotis — CHANGES_REQUESTED** [review]
Please create class structure

**Review by vvdanila — APPROVED** [review]

**Review by raicubogdan — APPROVED** [review]

**Comment by Crystiann** [comment]
# please make it simple

**Pull request merged**